src/wp-includes/class-wp-autoload.php

```diff
@@ -415,13 +415,14 @@ public static function register_core() {
 	public static function autoload_core( $class_name ) {
 		// Lowercase the classname to accomodate for WP classes written with wrong cases.
 		$class_name = strtolower( $class_name );
+		$paths      = static::CLASSES_PATHS;
 
 		// Bail early if the class is not a WP class.
-		if ( ! isset( static::CLASSES_PATHS[ $class_name ] ) ) {
+		if ( ! isset( $paths[ $class_name ] ) ) {
```

Comment:
Don't turn the static array into a variable again, this defeats the non-memory-allocation approach.
To get rid of the PHP error, try this instead:
```
if ( ! array_key_exists( $class_name, static::CLASSES_PATHS ) ) {
```

Reply:
Thank you for the insights! Fixed 👍